package.json

```diff
@@ -45,6 +45,7 @@
     "eslint-plugin-import": "2.27.5",
     "eslint-plugin-prettier": "4.2.1",
     "eslint-plugin-unicorn": "47.0.0",
+    "parse-duration": "1.0.3",
```

Comment:
`parse-duration` should be in `dependencies`

> If it absolutely have to, you should add to `devDependencies` becasuse it doesn't need in runtime.

for this I means `@types/...` packages, sorry for ambiguity.